mod/wet4/views/default/page/elements/head.php

```diff
@@ -95,149 +101,138 @@
-          }
-
-          $pubDate = date ("Y-m-d", elgg_get_excerpt($my_page_entity->time_created));
-          $lastModDate = date ("Y-m-d", elgg_get_excerpt($my_page_entity->time_updated));
-
-          $datemeta = '<meta name="dcterms.issued" title="W3CDTF" content="' . $pubDate . '"/>';
-          $datemeta .= '<meta name="dcterms.modified" title="W3CDTF" content="' . $lastModDate . '" />';
-      } else {
-          $desc = $vars['title'];
-          $briefdesc = $vars['title'];
-      }
-
-      $creator = gc_explode_translation(elgg_get_page_owner_entity()->name,get_current_language());
-
-      if(!$creator){
-          $creator = 'GCconnex';
-      }
-
-
-		// cyu - prevent crawler to index unsaved draft
-		if ($my_page_entity instanceof ElggObject) {
-			if ($my_page_entity->getSubtype() === 'blog' && strcmp($my_page_entity->status,'unsaved_draft') == 0)
-				echo '<meta name="robots" content="noindex">';
-		}
-
-          $no_index_array = array(
-            'activity','activity/all','/activity/owner','/activity/friends/','/activity_tabs/mydept','/activity_tabs/otherdept',
-            'blog/all','blog/owner/','/blog/group/','/blog/friends/',
-            'bookmarks/all','bookmarks/owner/','/bookmarks/friends/','/bookmarks/group/',
-            'event_calendar/list',
-            'file/all','/file/owner/','/file/friends/',
-            'photos/all','photos/owner','photos/friends/',
-            'members','/members/popular/','/members/online','/members/department',
-            'polls/all','/polls/owner/','/polls/friends/',
-            'groups/all','/groups/owner/','/groups/invitation',
-            'photos/siteimagesowner/',
-            'thewire/all','/thewire/owner/','/thewire/friends/',
-            'file_tools/list', '/newsfeed/',
-          );
-
-          $can_index = true;
-          foreach ($no_index_array as $partial_url) {
-            // if url is found, dont index
-            if (strpos($_SERVER['REQUEST_URI'],$partial_url) !== false ) {
-              $can_index = false;
-              break;
-            }
-          }
-
-          if (!$can_index) {
-?>
+if ($my_page_entity) {
+
+  if (elgg_instanceof($my_page_entity, 'group')) {
+      $desc = elgg_strip_tags(elgg_get_excerpt(gc_explode_translation($my_page_entity->description,get_current_language())));
+      $briefdesc = gc_explode_translation($my_page_entity->briefdescription,get_current_language());
+  } else if (elgg_instanceof($my_page_entity, 'user')) {
+      $desc = elgg_echo('profile:title', array(gc_explode_translation($my_page_entity->name,get_current_language())));
+      $briefdesc = elgg_echo('profile:title', array(gc_explode_translation($my_page_entity->name,get_current_language())));
+  } else {
+      $desc = gc_explode_translation($my_page_entity->title,get_current_language());
+      $briefdesc = gc_explode_translation($my_page_entity->title,get_current_language());
```

Comment:
I'll add that in the next update (along with a small minor changes), I just want this to be pushed out